src/modules/dailyCreditsSummary/i18n/pl_PL.ts

```diff
@@ -0,0 +1,106 @@
+export default {
+    amount: 'Amount',
```

Comment:
Amount = Kwota?

Reply:
yes

Reply:
Kwota (money) or Ilość (how many/much)

Reply:
Do you want to change it then or should it stay as "amount"?

Reply:
It should be changed